ios/FluentUI/Label/Label.swift

```diff
@@ -67,8 +71,8 @@ open class Label: UILabel, TokenizedControlInternal {
     }
 
     open override var attributedText: NSAttributedString? {
-      didSet {
-          isUsingCustomAttributedText = attributedText != nil
+        didSet {
+            isUsingCustomAttributedText = attributedText != nil
```

Comment:
Can you make sure `attributedText` still works? I think `TableViewCell` uses `Label`s with attributed strings.

Reply:
Yep, attributedText looks good.

<img src="https://user-images.githubusercontent.com/55368679/231876158-bd7c4655-74e8-46e8-9f62-a74b95665bb7.png"  width="300">